lifecycle/lifecycle/job/deployment.py

```diff
@@ -119,6 +119,17 @@ def save_deployment_phase(deployment_id: str, phase: str):
     deployment.save()
 
 
+@db_access
+def save_deployment_warnings(deployment_id: str, warnings: str):
+    try:
+        deployment = models.Deployment.objects.get(id=deployment_id)
+    except models.Deployment.DoesNotExist:
+        raise EntityNotFound(f'deployment with ID {deployment_id} was not found')
+    deployment.warnings = f'{deployment.warnings},\n{warnings}' if deployment.warnings else warnings
```

Comment:
Maybe let's keep just one character as a delimiter for simplicity and easier parsing. Plus, warning sentence might end with a dot, and then it would look stupid.
```suggestion
    deployment.warnings = f'{deployment.warnings}\n{warnings}' if deployment.warnings else warnings
```

Reply:
Good catch.